<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPortConfigPanel</class>
 <widget class="QWidget" name="SerialPortConfigPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBox_Parity">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comPortBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>停止位</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_DateBits">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_BaudRate">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_StopBits">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>数据位</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>波特率</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>奇偶校验</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>串口号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
